skimage/filters/rank/generic.py

```diff
@@ -23,13 +23,12 @@
 image) where non zero values are the part of the image participating in the
 histogram computation. By default the entire image is filtered.
 
-This implementation outperforms grey.dilation for large structuring elements.
+This implementation outperforms gray.dilation for large structuring elements.
```

Comment:
What does `gray.dilation` refer to here?

Reply:
Maybe `gray_dilation`?

Reply:
Actually this refers to the `dilation` function in `grey.py`, so the new wording is incorrect, unfortunately.

Reply:
Probably an outdated comment, meaning `morphology.dilation`.